shared/tests/utility/motion/TestSplines.cpp

```diff
@@ -19,110 +19,111 @@
-                                        {12, 10, 66},
-                                        {1, 0, 1},
-                                        {11, 4, 330},
-                                        {1, 1, 1},
-                                        {33, 2, 528},
-                                        {22, 5, 26334},
-                                        {43, 21, 1052049481860},
-                                        {13, 6, 1716},
-                                        {9, 6, 84},
-                                        {41, 21, 269128937220},
-                                        {6, 1, 6},
-                                        {7, 3, 35},
-                                        {4, 3, 4},
-                                        {24, 24, 1},
-                                        {11, 2, 55},
-                                        {23, 23, 1},
-                                        {35, 18, 4537567650},
-                                        {6, 4, 15},
-                                        {27, 1, 27},
-                                        {6, 2, 15},
-                                        {21, 16, 20349},
-                                        {24, 12, 2704156},
-                                        {38, 32, 2760681},
-                                        {46, 16, 991493848554},
-                                        {13, 1, 13},
-                                        {11, 0, 1},
-                                        {28, 26, 378},
-                                        {44, 5, 1086008},
-                                        {21, 20, 21},
-                                        {39, 23, 37711260990},
-                                        {27, 6, 296010},
-                                        {12, 8, 495},
-                                        {22, 18, 7315},
-                                        {27, 2, 351},
-                                        {9, 9, 1},
-                                        {9, 6, 84},
-                                        {10, 7, 120},
-                                        {25, 18, 480700},
-                                        {22, 3, 1540},
-                                        {48, 2, 1128},
-                                        {50, 20, 47129212243960},
-                                        {37, 21, 12875774670},
-                                        {24, 6, 134596},
-                                        {15, 1, 15},
-                                        {39, 21, 62359143990},
-                                        {44, 20, 1761039350070},
-                                        {21, 21, 1},
-                                        {14, 11, 364},
-                                        {43, 0, 1},
-                                        {1, 0, 1},
-                                        {39, 21, 62359143990}};
+std::vector<std::vector<size_t>> C = {{3, 0, 1},
```

Comment:
Note that these ones testing the combination should be `size_t`, because the `Combination` functions take `size_t` arguments. Fixing that is beyond the scope of this PR